src/Kinto.elm

```diff
@@ -529,10 +513,11 @@ withQueryParam param builder =
 
         queryString =
             params
-                |> List.append [ param ]
-                |> joinUrlEncoded
+                |> List.append [ string paramKey paramValue ]
+                |> toQuery
+                |> String.replace "%20" "+"
```

Comment:
Why are you replacing the `%20` by `+`? Is it needed by kinto?

Reply:
I don't think it is but the URL is easier to read.